add docs for enforce mpp (#5811)
pingcap/docs: tidb-configuration-file.md

+ Default value: `20971520`
+ When `nested-loop-join-cache-capacity` is set to `0`, nested loop join cache is disabled by default. When the LRU size is larger than the value of `nested-loop-join-cache-capacity`, the elements in the LRU are removed.

### `enforce-mpp`

> + Determines whether to ignore the optimizer cost estimation and force TiFlash's MPP mode to execute queries.

Reviewer: ```suggestion
+ Determines whether to ignore the optimizer's cost estimation and to force queries to be executed using TiFlash's MPP mode.
```

> + Determines whether to ignore the optimizer cost estimation, and force TiFlash's MPP mode to execute queries.

Reviewer 2: and to force?

> + Determines whether to ignore the optimizer's cost estimation and to force queries to be executed using TiFlash's MPP mode.

Reviewer: ```suggestion
+ Determines whether to ignore the optimizer's cost estimation and to forcibly use TiFlash's MPP mode for query execution.
```
